### PR TITLE
Upgrade parent to 14.10

### PR DESCRIPTION
**Pull request by Farcasut**
Updated the parent of the application to XWiki 14.10. With this upgrade, the custom  svg handler has become obsolete since XWiki now uses a new method for exporting a pdf document, which eliminates the need for custom handling.

### CODE REVIEW

File: application-diagram-ui/src/main/resources/Diagram/DiagramConfig.xml [hunk 1]
```diff
@@ -318,9 +318,6 @@
     <property>
       <configurationClass/>
     </property>
-    <property>
-      <configureGlobally>1</configureGlobally>
```
Comment (oanalavinia) [inline application-diagram-ui/src/main/resources/Diagram/DiagramConfig.xml:322]: If this property was deleted, wasn't it replaced by something else?

File: application-diagram-ui/src/main/resources/Diagram/DiagramEditSheet.xml [hunk 2]
```diff
@@ -1056,66 +544,8 @@ define('diagram-link-editor', [
...
           str = $('&lt;div/&gt;').html(str).text() || this.foAltText;
         }
-        // Update links inside foreignObject to use absolute URLs, since the generated SVG needs to be portable.
-        $(fo).find('a').each(function() {
-          var oldLink = $(this).attr('href');
-          if (diagramLinkHandler.isXWikiCustomLink(oldLink)) {
-            var newLink = diagramLinkHandler.getURLFromCustomLink(oldLink);
-            $(this).attr('href', newLink);
-          }
-        });
```
Comment (oanalavinia) [inline application-diagram-ui/src/main/resources/Diagram/DiagramEditSheet.xml:1066]: Are the links of the new PDF export absolute?

File: pom.xml [hunk 3]
```diff
@@ -173,8 +173,8 @@
     </plugins>
   </build>
   <properties>
-    <draw.io.version>24.5.5</draw.io.version>
-    <licensing.version>1.25</licensing.version>
```
Comment (oanalavinia) [inline pom.xml:177]: I don't think these should be changed in this PR

**Review by oanalavinia — CHANGES_REQUESTED** [review]
Please check also the merge conflicts

File: application-diagram-ui/src/main/resources/Diagram/DiagramEditSheet.xml (hunk 2 above)
Comment (Farcasut) [reply application-diagram-ui/src/main/resources/Diagram/DiagramEditSheet.xml:1066]: After some testing, it appears this code is never called, and the links in the XWiki export are not displayed at all. Do you think I should remove this code now, or wait until I actually fix the issue?

File: application-diagram-ui/src/main/resources/Diagram/DiagramConfig.xml (hunk 1 above)
Comment (Farcasut) [reply application-diagram-ui/src/main/resources/Diagram/DiagramConfig.xml:322]: Hi! I tried exporting the page again, but the property was not replaced with anything.

Comment (Farcasut) [reply application-diagram-ui/src/main/resources/Diagram/DiagramConfig.xml:322]: The property is already present:
https://github.com/xwikisas/application-diagram/blob/462f79ac0e95183abf63801f5c99567b09ddfe5c/application-diagram-ui/src/main/resources/Diagram/DiagramConfig.xml#L343

**Pull request merged**